introduction.md

```diff
@@ -1,2 +1,5 @@
 This is my introduction. I am Jay Dreyer. Hi, Dojo!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?